IRremoteESP8266.cpp

```diff
@@ -423,18 +437,16 @@ void IRsend::sendSharpRaw(unsigned long data, int nbits) {
         space(SHARP_ZERO_SPACE);
       }
     }
-    
     mark(SHARP_BIT_MARK);
     space(SHARP_ZERO_SPACE);
     delay(40);
-
-    data = data ^ SHARP_TOGGLE_MASK;
+    data = data ^ SHARP_TOGGLE_MASK;  // invert data
   }
 }
 
 // Sharp send compatible with data obtained through decodeSharp
-void IRsend::sendSharp(unsigned int address, unsigned int command) {
-  sendSharpRaw((address << 10) | (command << 2) | 2, 15);
+void IRsend::sendSharp(unsigned int address, unsigned int extended) {
+  sendSharpRaw((address << 10) | (extended << 2) | 2, 15);
```

Comment:
I'm not familiar with Sharp codes, but is it really necessary to rename the parameter?